// Proto file representing references to EDB Object classes

syntax = "proto3";

package ansys.api.edb.v1;

import "edb_messages.proto";

message LayerRefMessage {
  oneof ref {
    EDBObjMessage id = 1;
    google.protobuf.StringValue name = 2;
  }
}

message NetRefMessage {
  oneof ref {
    EDBObjMessage id = 1;
    google.protobuf.StringValue name = 2;
  }
}
